linear_algebra/code/CMakeLists.txt

```diff
@@ -3,11 +3,20 @@ project(wg21_linear_algebra VERSION 0.0.1)
 
 option(BUILD_TESTING "flag to build tests" ON)
 
+include(FetchContent)
```

Comment:
Fetch Content was introduced in CMake 3.11.  This was one of the reasons I wanted to upgrade to a recent CMake version.  As discussed we could get the MDSpan dependency in numerous ways, Fetch Content and Conan being the 2 easily accessible to the project's current set up.

MDSpan, like the Linear Algebra library, is header-only which means they are both exposed as "INTERFACE" libraries.  Conan excels at pulling in pre-built binaries for a wide range of configurations (os, arch, configuration, compiler, etc), however, given LA and MDSpan are header-only Fetch_Content is sufficient and the simplest route for now. 

Fetch Content will pull a copy of the MDSpan repo into the build directory in `<build dir>/_dep/mdspan-src`.  Thankfully MDSpan is correctly set up to be included via this mechanism and correctly supports this via an interface and alias target.